src/bp/common/variables.ts

```diff
@@ -81,4 +85,8 @@ export class BaseVariable<T, V = any> implements BoxedVariable<T, V> {
       confidence: this._confidence!
     }
   }
+
+  parse(text: string): T {
+    return <any>text
```

Comment:
This is a part of the code I know nothing about, but this just feels wrong.

variable.parse("Some text") returns "Some text" casted to any? 

Shouldnt this do some actual parsing or something?

Reply:
This is the base class. The default behavior is that we do nothing. Other derived class do actual parsing here

Reply:
:+1: